src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.module.scss

```diff
@@ -0,0 +1,314 @@
+@import '../Media/Media.scss';
+@import '../Variables.scss';
+
+$tablet-small-end: 670px;
+
+.flex {
+  display: flex;
+}
+
+.itemsCenter {
+  align-items: center;
+}
+
+.hamburger {
+  position: fixed;
+  top: 24px;
+  right: 24px;
+  z-index: 2;
+
+  @include for-laptop-and-up {
+    display: none;
+  }
+}
+
+.phoneAndTablet {
+  position: relative;
+  height: var(--Space-64);
+  background-color: white;
+  padding: var(--Space-24) var(--ContainerPadding--Phone);
+  display: none;
+  align-items: center;
+  justify-content: flex-end;
+  display: flex;
+  @include for-laptop-and-up {
+    display: none;
+  }
+  &.visible {
+    z-index: 3;
```

Comment:
Can we store some z-index variables in Variables.scss? 
Is there a reason we have multiple z-indexes?

Reply:
it's a bit of a workaround for the hamburger toggle, i'll move to variables

Reply:
marked in TODOs for fastfollow